### PR TITLE
intractiv chart

### PR DESCRIPTION
**Pull request by Tanzorr**

### CODE REVIEW

File: frontend/src/components/Projects/StatusReports/Create/StatusReport.vue [hunk 1]
```diff
@@ -347,7 +350,6 @@
                 if (!this.editable) {
                     return;
                 }
-
```
Comment (maxlenash) [inline frontend/src/components/Projects/StatusReports/Create/StatusReport.vue:350]: please return this empty row

File: frontend/src/components/Projects/StatusReports/Create/StatusReport.vue [hunk 2]
```diff
@@ -361,7 +363,10 @@
...
```
Comment (maxlenash) [inline frontend/src/components/Projects/StatusReports/Create/StatusReport.vue:364]: please return thisempty row

File: frontend/src/components/Projects/StatusReports/Create/StatusReport.vue [hunk 3]
```diff
@@ -361,7 +363,10 @@
...
+                let colors = [themes.light.lightRed, themes.light.lightYellow, themes.light.lightGreen];
+                this.trendChartData[3] =value;
```
Comment (maxlenash) [inline frontend/src/components/Projects/StatusReports/Create/StatusReport.vue:367]: should be "= value"

File: frontend/src/components/Projects/StatusReports/Create/StatusReport.vue (hunk 2 above)
Comment (Tanzorr) [reply frontend/src/components/Projects/StatusReports/Create/StatusReport.vue:364]: ok

File: frontend/src/components/Projects/StatusReports/Create/StatusReport.vue (hunk 1 above)
Comment (Tanzorr) [reply frontend/src/components/Projects/StatusReports/Create/StatusReport.vue]: ok

File: frontend/src/components/Projects/StatusReports/Create/StatusReport.vue (hunk 3 above)
Comment (Tanzorr) [reply frontend/src/components/Projects/StatusReports/Create/StatusReport.vue]: ok

**Pull request merged**